io/disk/arcconf/arcconf_raid_oper.py.data/Readme

```diff
@@ -0,0 +1,14 @@
+arcconf - Command line interface to configure PMC-Sierra
+(Microsemi) Controllers.
+
+arcconf_raid_oper.py covers functionality like RAID creation,
+deletion, Physical disk & logical disk format, and gather
+controller status using getconfig.
```

Comment:
I retained the readme text , since thats where most of the testers look for first hand information.

instead changed the wording for class doc string to below:
    Covers functionality  relavent to RAID operations like creation,
    deletion, Physical logical disk format and status information.